Cargo.toml

```diff
@@ -21,6 +21,8 @@ aes-gcm = "0.10"
 aes = "0.8"
 sha2 = "0.10"
 md-5 = "0.10"
+anyhow = "1.0.86"
+reqwest = "0.12.5"
```

Comment:
why do we need reqwest here? can't we use the cf worker's [request](https://docs.rs/worker/latest/worker/struct.Request.html) instead?

Reply:
it seems you can't just send raw byte slices. I found a hack to construct the JsValue but i decided to just use reqwest because it was official recommenced and was easier.